lcUI/widgets/clicommand.h

```diff
@@ -54,6 +54,7 @@ class CliCommand : public QDockWidget {
          * Disables text parsing.
          */
 		void returnText(bool returnText);
+        double lctolerance = 1e-10;
```

Comment:
You should use the value from settings class. Duplicating it can cause inconsistencies if it's updated only in settings class